pkg/cue/model/value/value.go

```diff
@@ -23,19 +23,17 @@ import (
 	"strconv"
 	"strings"
 
-	"cuelang.org/go/cue/token"
-
-	"github.com/oam-dev/kubevela/pkg/stdlib"
-
 	"cuelang.org/go/cue"
 	"cuelang.org/go/cue/ast"
 	"cuelang.org/go/cue/build"
 	"cuelang.org/go/cue/literal"
 	"cuelang.org/go/cue/parser"
+	"cuelang.org/go/cue/token"
 	"github.com/pkg/errors"
 
 	"github.com/oam-dev/kubevela/pkg/cue/model/sets"
 	"github.com/oam-dev/kubevela/pkg/cue/packages"
+	"github.com/oam-dev/kubevela/pkg/stdlib"
```

Comment:
FYI, I was told by a Crossplane guy that package names like `util` or `stdlib` are bad. You can refer to `Bad package name` section of https://go.dev/blog/package-names:)